impl/client/src/main/javascript/web/pentaho/visual/config.js

```diff
@@ -834,7 +834,9 @@ define(function() {
         },
         apply: {
           extension: {
-            valuesVisible: true
+            valuesVisible: true,
+            legendAreaVisible: true,
```

Comment:
Why is this property needed? The visibility of the legend is already controlled by the base class' `Abstract#showLegend`, and this in turn is controlled by Analyzer chart options.